reflex/components/component.py

```diff
@@ -467,10 +475,10 @@ def render(self) -> Dict:
             tag.add_props(
                 **self.event_triggers,
                 key=self.key,
-                sx=self.style,
                 id=self.id,
                 class_name=self.class_name,
                 **self.custom_attrs,
+                **self._get_style(),
```

Comment:
```suggestion
                **self._get_style(),
                **self.custom_attrs,
```

lets flip these around, so that `custom_attrs` always takes precedence

Reply:
got it